astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -711,12 +714,20 @@ def test_aa_high_precision():
 
     # Numbers from
     # https://github.com/astropy/astropy/pull/11073#issuecomment-735486271
-    TARGET_AZ, TARGET_EL = 15.032673509886*u.deg, 50.303110133928*u.deg
+    # updated in https://github.com/astropy/astropy/issues/11683
+    TARGET_AZ, TARGET_EL = 15.032673509956*u.deg, 50.303110133923*u.deg
     TARGET_DISTANCE = 376252883.247239*u.m
-
-    assert_allclose(moon_aa.az, TARGET_AZ, atol=19.62*u.uas, rtol=0)
-    assert_allclose(moon_aa.alt, TARGET_EL, atol=0.85*u.uas, rtol=0)
-    assert_allclose(moon_aa.distance, TARGET_DISTANCE, atol=38.87*u.mm, rtol=0)
+    # Pre-2.15, some versions of jplephem suffer from time rounding errors.
+    # We check whether we are up to date by looking for the entry in the
+    # changelog, which is included in the docstring.
+    if '**2020 September 2 — Version 2.15**' in jplephem.__doc__:
```

Comment:
The problem with doing this is that is jplephem is ever updated to 2.16, the test will revert to the higher tolerances.

Reply:
No, the docstring contains entries for all versions, so in that sense it is OK. But I think it should be rewritten to use the version information regardless, as suggested in https://github.com/astropy/astropy/issues/11683#issuecomment-832368203 (will have to see if it can be done without `pkg_resources`)

Reply:
Oh, OK. I didn't realize that.

Reply:
Why is `jplephem.__version__` not a thing? 😱 

Is the version embedded in `__doc__` predictable? If so, we can probably extract it with regex and then use the normal version comparison. I think we can `xfail` for 2.14 instead of adding an `if-else`?